### PR TITLE
Minor updates to Ubuntu setup instructions

### PR DESCRIPTION
**Pull request by balderdash**
Updated expected version numbers for npm/node, added a command to
set the default ruby version for rvm users.

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
 ## Ubuntu 14.04
 
 1. `sudo apt-get update`
-1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-7-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk libsqlite3-dev phantomjs`
+1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-7-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk libsqlite3-dev phantomjs build-essential`
```
Comment (Hamms) [inline SETUP.md:45]: good catch!

File: SETUP.md [hunk 2]
```diff
@@ -62,11 +62,15 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
     - C. `rvm`. A few folks have had more luck with rvm vs rbenv on linux.
         1. Install rvm from https://rvm.io/
         1. `rvm install 2.2.3`
+        1. `rvm use 2.2.3 --default`
 1. Install Node.js 0.12.4 and npm 2.10.1
-  1. Option A - nodesource repository
+  1. Option A - nvm
+    1. `curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash`
+    1. nvm install 0.12.4
```
Comment (Hamms) [inline SETUP.md]: minor nitpick; this line should probably be formatted as inline code

**Comment by Hamms** [comment]
One minor nitpick that you can take or leave at your discretion; otherwise, LGTM!

**Comment by Hamms** [comment]
grats on your first PR :D

File: SETUP.md (hunk 2 above)
Comment (balderdash) [inline SETUP.md]: Done.

**Pull request merged**